portal/templates/profile_macros.html

```diff
@@ -510,24 +515,26 @@ <h4 class="text-success registration-label">{{_('registered')}}</h4>
 {% macro profileOrgsSelector(person, current_user, consent_agreements) -%}
     <div id="userOrgs_view" class="view-container"></div>
     <div class="edit-container">
-        <div id="userOrgs" class="customized">
-           <input type="hidden" id="stock_consent_url" value="{{ url_for('portal.stock_consent', org_name='placeholder', _external=True)}}" />
-           <p class="prompt text-muted">{{ _("In what state ")}}{% if current_user and current_user.has_role(ROLE.STAFF)%}{{_("is the patient ")}}{%else%}{{_("are you ")}}{% endif %}{{_("currently receiving prostate cancer care?") }}</p>
-           <div class="state-selector-container selector-show"><select id="stateSelector" class="form-control"></select></div>
-           <div id="fillOrgs" userId="{{person.id if person}}" class="orgs-by-state" {%if person and person.has_role(ROLE.PATIENT) and person.id == current_user.id %}patient_view="true"{%endif%}>
-               <p class="clinic-prompt prompt selector-show">{{ _("What is your main clinic for prostate cancer care?")}}</p>
-               <div class="main-state-container selector-show">
+        <div id="clinics">
+            <div id="userOrgs" class="customized">
+               <input type="hidden" id="stock_consent_url" value="{{ url_for('portal.stock_consent', org_name='placeholder', _external=True)}}" />
+               <p class="prompt text-muted">{{ _("In what state ")}}{% if current_user and current_user.has_role(ROLE.STAFF)%}{{_("is the patient ")}}{%else%}{{_("are you ")}}{% endif %}{{_("currently receiving prostate cancer care?") }}</p>
```

Comment:
You'll likely need to split this into two complete sentences for translation (see below). Breaking up a sentence into multiple translation calls makes it impossible to translate (each string is translated individually).
```
{% if staff %}
__("Sentence A")
{% else %}
__("Sentence B")
{% endif %}
```